rustls/src/client/client_conn.rs

```diff
@@ -193,6 +193,14 @@ pub struct ClientConfig {
     /// The default is false.
     pub enable_early_data: bool,
 
+    /// If set to `true`, requires the server to support the extended
+    /// master secret extraction method defined in [RFC 7627].
+    /// The default is `false`.
```

Comment:
Maybe add a note here about the FIPS connection?

Reply:
Added